DeepSpeech.py

```diff
@@ -969,6 +969,10 @@ def done(self):
 
                 self.loss = agg_loss / num_jobs
 
+                # if the job was for validation dataset then append it to the COORD's _loss for early stop verification
+                if (FLAGS.early_stop is True) and (self.set_name == 'dev'):
+                    COORD._dev_losses.append(self.loss)
```

Comment:
Much better now - albeit you could also access loss values of the entries in `COORD._epochs_done` by some list comprehension. Your choice.